ganttproject/src/biz/ganttproject/impex/csv/CommonWriter.java

```diff
@@ -0,0 +1,15 @@
+package biz.ganttproject.impex.csv;
+
+import java.io.Closeable;
+import java.io.Flushable;
+import java.io.IOException;
+
+/**
+ * @author akurutin on 04.04.2017.
+ */
+public interface CommonWriter extends Flushable, Closeable {
+    public void print(Object value) throws IOException;
+    public void println() throws IOException;
+    public void close() throws IOException;
```

Comment:
`flush` and `close` are probably redundant too, as they are defined in the extended interfaces?